src/keyboard-shortcuts.tsx

```diff
@@ -0,0 +1,21 @@
+import { t } from './services';
+
+
+
+const shortcutItem = (labelKey: string, shortcut: string) => {
+  return {
+    label: t(labelKey),
+    shortcut
+  }
+};
+
+export const commandsGeneral = [
+  shortcutItem('commands-label.create', 'Option + n'),
+  shortcutItem('commands-label.edit', 'cmd + Option + e'),
+  shortcutItem('commands-label.save', 'cmd + s'),
+  shortcutItem('commands-label.duplicate', 'Option + C'),
+  shortcutItem('commands-label.toggle-lists-detail-accordion', 'spacebar'),
+  shortcutItem('commands-label.expand-list-detail-accordions', 'cmd  + Option + b'),
+  shortcutItem('commands-label.collapse-list-detail-accordions', 'cmd  + Option + g'),
+  shortcutItem('commands-label.go-to-filter-pane', 'cmd + s'),
```

Comment:
Do these shortcuts follow the key name conventions used in other parts of folio (e.g., cmd vs command vs Command, opt vs option vs Option, n vs N, etc)

I'm assuming this is okay, but just want to double check :)

Reply:
I looked around a bit, and it looks like other apps capitalize the first letter in "cmd" and "spacebar" along with the single letters (N, E, S, etc)